app/assets/stylesheets/govuk_publishing_components/components/_chat-entry.scss

```diff
@@ -3,6 +3,8 @@
 .gem-c-chat-entry {
   display: flex;
   align-items: flex-start;
+  background-color: #e8f1f8; // there is no Design System color palette option or Sass variable defined for #e8f1f8
```

Comment:
How would you feel about using a `govuk-tint` sass function here? I had a bit of a play around with things and it looks like if we do `background-color: govuk-tint(govuk-colour("blue"), 90%);`, we'll get `#e8f1f8`.

Reply:
Nice, thanks, I didn't think about the `govuk-tint` mixin 👍

Reply:
No worries 👍 I reckon we could remove the comment as well now (apols for the nitpicks, will approve if I'm getting too nitpicky!)

Reply:
No problem, I've removed it